setup.cfg

```diff
@@ -39,8 +39,8 @@ install_requires =
 spacy_factories =
     llm = spacy_llm.pipeline.llm:make_llm
 spacy_misc =
-    spacy.FewShotReader.v1 = spacy_llm.registry:fewshot_reader
-    spacy.FileReader.v1 = spacy_llm.registry:file_reader
+    spacy.FewShotReader.v1 = spacy_llm.registry.fewshot_reader
+    spacy.FileReader.v1 = spacy_llm.registry.file_reader
```

Comment:
Was this change just temporary to test the test suite?

Reply:
Yes. Will reverse.